jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/Stack.kt

```diff
@@ -45,11 +44,9 @@ private val UPDATE_STACK_STATUS_INTERVAL = Duration.ofSeconds(5)
 private val UPDATE_STACK_STATUS_INTERVAL_ON_FINAL_STATE = Duration.ofSeconds(60)
 private const val REDRAW_ANIMATED_ICON_INTERVAL = 70
 private const val TREE_TABLE_INITIAL_PROPORTION = 0.25f
-internal val STACK_TOOL_WINDOW =
-    ToolkitToolWindowType("AWS.CloudFormation", message("cloudformation.toolwindow.label"), icon = AwsIcons.Logos.CLOUD_FORMATION_TOOL)
 
 class StackWindowManager(private val project: Project) {
-    private val toolWindow = ToolkitToolWindowManager.getInstance(project, STACK_TOOL_WINDOW)
+    private val toolWindow = CloudFormationToolWindow.getOrCreateToolWindow(project)
```

Comment:
Can we just grab this each time, I'd rather release the memory back if all stack tabs are closed